test/DynamoCoreTests/CustomNodes.cs

```diff
@@ -765,20 +765,69 @@ public void TestCustomNodeDefaultValueJson()
 
             // check first input defaults
             string concreteType = (string)jObject.SelectToken("Nodes[1].ConcreteType");
-            string inputSymbol = (string)jObject.SelectToken("Nodes[1].InputSymbol");
+            string name = (string)jObject.SelectToken("Nodes[1].Parameter.Name");
+            string typeName = (string)jObject.SelectToken("Nodes[1].Parameter.TypeName");
+            int typeRank = (int)jObject.SelectToken("Nodes[1].Parameter.TypeRank");
             string defaultValue = (string)jObject.SelectToken("Nodes[1].Parameter.DefaultValue");
 
             Assert.IsTrue(concreteType == "Dynamo.Graph.Nodes.CustomNodes.Symbol, DynamoCore");
-            Assert.IsTrue(inputSymbol == "center: Point = Point.ByCoordinates(10,10,10);");
```

Comment:
why get rid of this - shouldn't the input symbol still be this string?

Reply:
Becase `inputSymbol` is no longer serialized in json like we did in xml, this assert will fail because the Json object does not contain `inputSymbol` property.